### ISSUE TITLE
[FR] `xr`/`xr-hyper` support for `ltproperties`

### ISSUE THREAD

**Issue by gusbrs** [latex3/latex2e#1180]
## Brief outline of the enhancement

This is just a suggestion / feature request. It would be really nice to have `xr`/`xr-hyper` support for `ltproperties`, so that `\new@label@record` data could be imported from an external document, with a corresponding `url`/`file` property.

## Minimal example showing the current behaviour

File `A.tex`:

```latex
\RequirePackage{latexbug}
\documentclass{article}

\begin{document}

\section{Section 1}
\label{foolabel}
\RecordProperties{ltp:foolabel}{label,page}

\end{document}
```
File `B.tex`:

```latex
\RequirePackage{latexbug}
\documentclass{article}

\usepackage{xr}
\externaldocument[A-]{A}

\begin{document}

\ref{A-foolabel}

\RefProperty{A-ltp:foolabel}{label}

\end{document}
```
Produces:
![Screenshot from 2023-11-14 08-06-23](https://github.com/latex3/latex2e/assets/19410606/b1d37ef7-4c09-4326-ade4-e1518c994d21)

In other words, `\ref` works for a label from the external document, while `\RefProperty` is not able to access the external document's label data.

## Minimal example showing the desired new behaviour

Well, it would be nice if it worked. For both `xr` and `xr-hyper` + `hyperref`.

**Comment by u-fischer** [comment]
The packages has been merged and with the next release both packages will support ltproperties.

**Comment by gusbrs** [comment]
@u-fischer and @davidcarlisle Thank you very much!
